CHANGELOG.md

```diff
@@ -1,3 +1,72 @@
+## 2022-10-06 v0.12.0
+
+Release highlights:
+- The performance and stability of the message `store` has improved dramatically. Query durations, even for long-term stores, have improved by more than a factor of 10.
+- Support for Waku Peer Exchange - a discovery method for resource-restricted nodes.
+- Messages can now be marked as "ephemeral" to prevent them from being stored.
+- [Zerokit](https://github.com/vacp2p/zerokit) is now the default implementation for spam-protected `relay` with RLN.
+
+The full list of changes is below.
+
+### Features
+
+- Default support for [Zerokit](https://github.com/vacp2p/zerokit) version of [`17/WAKU-RLN-RELAY`](https://rfc.vac.dev/spec/17/) implementation.
```

Comment:
```suggestion
- Default support for [Zerokit](https://github.com/vacp2p/zerokit) version of [`17/WAKU2-RLN-RELAY`](https://rfc.vac.dev/spec/17/) implementation.
```
The naming is fixed in an unmerged PR on the RFC repo, we can leave it as is in the release notes if you'd like!